src/apps/cif2bcif/index.ts

```diff
@@ -28,9 +45,20 @@ parser.addArgument([ 'src' ], {
 parser.addArgument([ 'out' ], {
     help: 'Output BCIF path'
 });
+parser.addArgument([ '-config' ], {
```

Comment:
I usually do a short '-c' and a long '--config' argument version

Reply:
Done in 956fc60